.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,64 @@
+name: CI/CD
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+
+env:
+    CI: true
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Use Node.js 18
```

Comment:
Please update to `20.x` and `actions/setup-node@v4`

Reply:
And the same in the `release` job.

Reply:
Done